src/styles/styles.js

```diff
@@ -522,19 +524,21 @@ const styles = {
         ],
     }),
     expensiTextInput: {
+        ...spacing.pv0,
         fontFamily: fontFamily.GTA,
         fontSize: variables.fontSizeNormal,
+        lineHeight: variables.fontSizeNormalHeight,
         color: themeColors.text,
         height: '100%',
-        paddingTop: 25,
+        paddingTop: 23,
         paddingBottom: 8,
         paddingHorizontal: 11.5,
```

Comment:
This should just be 11 and not 11.5

Reply:
Ah sorry for the confusion. We want 11 on each side and not 11.5. Thanks for fixing!

Reply:
I think you forgot about this one @parasharrajat